server/sonar-server/src/test/java/org/sonar/server/duplication/ws/ShowActionTest.java

```diff
@@ -134,8 +131,7 @@ public void no_duplications_when_no_data() throws Exception {
     ComponentDto componentDto = new ComponentDto().setId(10L);
     when(componentDao.getNullableByKey(session, componentKey)).thenReturn(componentDto);
 
-    MeasureKey measureKey = MeasureKey.of(componentKey, CoreMetrics.DUPLICATIONS_DATA_KEY);
-    when(measureDao.getNullableByKey(session, measureKey)).thenReturn(null);
+    when(measureDao.findByComponentKeyAndMetricKey(session, componentKey, CoreMetrics.DUPLICATIONS_DATA_KEY)).thenReturn(null);
 
     WsTester.TestRequest request = tester.newGetRequest("api/duplications", "show").setParam("key", componentKey);
     request.execute();
```

Comment:
it doesn't concern directly my scope and if I have been faster, I would have done it, but not now